packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -155,11 +180,13 @@ class PropertiesView extends StatelessWidget {
                 PropertiesView.scaleFactorForVerticalLayout);
 
         return Scrollbar(
-          controller: scrollController,
+          controller: widget.scrollController,
           thumbVisibility: true,
           child: ListView.builder(
-            controller: scrollController,
-            itemCount: properties.length + layoutExplorerOffset,
+            controller: widget.scrollController,
+            itemCount:
+                (_sortedProperties.isEmpty ? 1 : _sortedProperties.length) +
```

Comment:
Nit: consider pulling this into a local variable.

Reply:
Done!